### ISSUE TITLE
GraphQL: Access child elements within query

### ISSUE THREAD

**Issue by gregorterrill** [craftcms/cms#4843]
Would be nice if there was a way to access child elements, like the CraftQL plugin's `children`. For example, this CraftQL query returned a single city and a list of its districts (which are child entries of the city entry in Craft):

```
query apiQuery ($slug: String) {
  entries(section:[cities], type:[CitiesCity], slug:$slug, limit:1 ) {
    id
    slug
    title,
    districts: children {
      ...on CitiesDistrict {
        id
        slug
        title
      }
    }
  }
}
```
I would like to be able to do the above in Craft 3.3 but there isn't a way to access the children of an element within a single query as far as I can tell. If it is already possible, can it be added to the documentation with a sample query?

**Comment by andris-sevcenko** [comment]
Seems like an oversight to me.

**Comment by andris-sevcenko** [comment]
By "children", we're talking about children in structures, right?

**Comment by andris-sevcenko** [comment]
Yeah, for sure. I was just making sure you're not wishing to fetch all the related entries in any of the content field.

**Comment by andris-sevcenko** [comment]
> If it is already possible, can it be added to the documentation with a sample query?

Still have to do this.

**Comment by andris-sevcenko** [comment]
Resolved in a0aa5391c1a16626273f683924909a0e95e20e3d
